manual/guides/courses.md

```diff
@@ -0,0 +1,108 @@
+because some classes are offered only in one half of the school year.
+The suggested sequence accounts for this and plans accordingly. For students
+who test out of any classes, it's important that you talk with your
+advisor and make sure there are no foreseeable conflicts.
+
+# Undergraduate Checklist
+
+
+```markdown
+- [ ] Looked at suggested course sequence
+- [ ] Plan your classes for the following semester(s)
+- [ ] Talk with your faculty advisor
+- [ ] Make any edits due to course load, class timing, etc.
+- [ ] Talk with Becky to release your registration hold
+- [ ] Check your time ticket for registration
+- [ ] Register during your registration period through Self Service
+```
+
+# Graduate Students
+
+Each graduate student in ARFC should have a folder in Google Drive shared
+between them and Prof. Huff. In that folder should be a Student Plan in the
+form of a spreadsheet.
+
+<img src="https://user-images.githubusercontent.com/393899/84724236-44cf4b80-af4d-11ea-9ac4-8ce729b72679.png" alt="example-spreadsheet" width="750"/>
+
+Before registration for the coming semester, which is around the end of April
+for the coming Fall semester or the beginning of November for the
+coming Spring semester, graduate students should fill out their course plan
+and discuss it with Prof. Huff. Use these links to find the course
+requirements for the
+[MS degree](http://catalog.illinois.edu/graduate/engineering/nuclear-plasma-radiological-engineering-ms/) and
+[PhD degree](http://catalog.illinois.edu/graduate/engineering/nuclear-plasma-radiological-engineering-phd/).
+
+Early Research Assistantship students should be taking 2-3 graduate classes
+per semester and 4-8 research credit hours. Later, students typically
+take one or fewer graduate classes per semester and fill the rest of their
+necessary hours with research credits.
+
+After Prof. Huff approves your course plan in the Student Record, you can
+request the registration hold be lifted so that you can go ahead and
+register for classes. To do so, send a polite email to the student office in
+NPRE (specifically, the graduate student advisor staff member in NPRE),
+CC'ing Prof. Huff, with a PDF copy of your student record document attached.
+Let the staff member know that we have discussed the attached course
+plan, Prof. Huff has approved it, and you would like to request the
+registration hold be lifted.
+
+Near the beginning of the semester, all students should update their Student
+Record with the grades they have earned in the courses from the past
+semester.
+
+# Graduate Checklist
+
+```markdown
+- [ ] Fill out your Student Plan
+- [ ] Discuss your selection with Prof. Huff
+- [ ] Make any edits due to course load, class timing, etc.
+- [ ] Request your registration hold lifted
+- [ ] Register for classes through Self Service
+- [ ] Register for Prof. Huff's Thesis Research using CRN 24115
+- [ ] Make sure you meet the credit requirements
```

Comment:
Please add a link here.

Reply:
Do you just want a link that talks about the 12 credit-hour requirement?